v2/tools/generator/internal/codegen/pipeline/remove_status_property_validations.go

```diff
@@ -114,3 +115,12 @@ func removeValidatedType(this *astmodel.TypeVisitor, vt *astmodel.ValidatedType,
 func removeEnumValidations(this *astmodel.TypeVisitor, et *astmodel.EnumType, _ interface{}) (astmodel.Type, error) {
 	return et.WithoutValidation(), nil
 }
+
+// removeKubebuilderRequired removes kubebuilder:validation:Required from all properties
+func removeKubebuilderRequired(this *astmodel.TypeVisitor, ot *astmodel.ObjectType, ctx interface{}) (astmodel.Type, error) {
```

Comment:
Isn't this going to remove it from ALL `ObjectType` instances - don't you only want to do this for Status types?

Reply:
If you look up in this file on line 57, we only run this on status types.

Reply:
All good then. 😁 #approved